mmv1/third_party/terraform/resources/resource_storage_bucket.go.erb

```diff
@@ -248,6 +248,23 @@ func resourceStorageBucket() *schema.Resource {
 				Description: `The bucket's Versioning configuration.`,
 			},
 
+			"autoclass": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
```

Comment:
Based on the API, it looks like this field might not need to be `Computed: true`. Does the server set this value if it is not provided by the user?